proxy/pkg/zdmproxy/response.go

```diff
@@ -37,3 +41,20 @@ func (r *Response) GetStreamId() int16 {
 		return r.requestFrame.Header.StreamId
 	}
 }
+
+type ResponseError struct {
```

Comment:
I'd probably add this type near the code of `cqlconnection` or the control connection because it really only applies there right? I don't think it is used outside of the control connection.

Reply:
I was actually hoping this type to be reusable when somebody needs to raise error and carry response frame. Therefore, I also tried to think of some reusable name.